sql/core/src/test/resources/sql-tests/results/window.sql.out

```diff
@@ -479,6 +479,38 @@ Anthony Bow	6627	Gerard Bondur
 Leslie Thompson	5186	Gerard Bondur
 
 
+-- !query
+SELECT
+    employee_name,
+    salary,
+    nth_value(employee_name, 2) OVER (
+      ORDER BY salary DESC
+      ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) second_highest_salary
+FROM
+    basic_pays
+ORDER BY salary DESC
+-- !query schema
+struct<employee_name:string,salary:int,second_highest_salary:string>
+-- !query output
+Larry Bott	11798	NULL
+Gerard Bondur	11472	Gerard Bondur
+Pamela Castillo	11303	Gerard Bondur
```

Comment:
Not related to this PR. We should fix the test framework so that the result is always aligned.

Reply:
The output comes from `hiveResultString`